pom.xml

```diff
@@ -172,6 +176,21 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <!-- https://github.com/wildfly/jandex-maven-plugin -->
+        <groupId>org.jboss.jandex</groupId>
+        <artifactId>jandex-maven-plugin</artifactId>
```

Comment:
What does this do? Documentation by project is quite sparse.
Is it still actively developed? On https://github.com/wildfly/jandex-maven-plugin it's marked as archived, but maybe it just moved to somewhere else.

Reply:
Added it when I had OpenAI client library used and native image creation was failing. Have removed this now.